data/json/items/comestibles/alcohol.json

```diff
@@ -1425,6 +1425,34 @@
     "fun": 25,
     "vitamins": [ [ "vitA", 2 ], [ "vitC", 23 ], [ "calcium", 1 ], [ "iron", 2 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "drink_bloody_mary_blood",
+    "name": { "str": "'exalted' bloody mary cocktail" },
```

Comment:
'classic' bloody mary cocktail Makes more sense as a naming scheme to me.
https://www.thedailymeal.com/bloody-mary-has-real-blood-it-3614

Reply:
Problem is that there is no blood in the original recipe, so there is no contrast or irony in using this term, in my eyes.
Change my mind?